org.osgi.test.junit4/src/test/java/org/osgi/test/junit4/service/FilterArgumentsTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) OSGi Alliance (2019). All Rights Reserved.
```

Comment:
All these new source files should just be `(2020)`.

Reply:
right right.. fixing